#! /usr/bin/env python

PACKAGE='jsk_perception'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

seed_method_enum = gen.enum([gen.const("absolute_seed", int_t, 0, "use absolute foreground/background pixel for seed mask"),
                             gen.const("probable_seed", int_t, 1, "use probable foreground/background pixel for seed mask")],
                             "seed pixel policy")
gen.add("seed_pixel_policy", int_t, 0, "policy of seed pixel",
        0, 0, 1,
        seed_method_enum);

exit(gen.generate(PACKAGE, "jsk_perception", "GrabCut"))
